package.json

```diff
@@ -49,9 +50,10 @@
     "emitter20": "^2.0.0",
     "emoji-strip": "^1.0.1",
     "expo": "^41.0.1",
-    "expo-clipboard": "^1.0.2",
+    "expo-clipboard": "1.0.2",
```

Comment:
If this dependency needs to be locked, please also add an entry and comment to the `reject` list in https://github.com/cybersemics/em/blob/dev/.ncurc.js. We use [ncu](https://github.com/raineorshine/npm-check-updates) to keep dependencies up-to-date: